concepticondata/conceptlists/Zhou-2020-350.tsv

```diff
@@ -0,0 +1,351 @@
+Zhou-2020-350-238	238	to escape	逃	1615	ESCAPE
+Zhou-2020-350-239	239	to exchange	交換	2509	EXCHANGE
+Zhou-2020-350-240	240	to exit	出	1107	GO OUT
+Zhou-2020-350-241	241	to expose under sun	曬		
+Zhou-2020-350-242	242	to fear	怕	1419	FEAR (BE AFRAID)
+Zhou-2020-350-243	243	to feed	餵	739	FEED
+Zhou-2020-350-244	244	to feed	飼養	739	FEED
+Zhou-2020-350-245	245	to flip over	翻過來		
+Zhou-2020-350-246	246	to float	浮	1574	FLOAT
+Zhou-2020-350-247	247	to fly	飛	1441	FLY (MOVE THROUGH AIR)
+Zhou-2020-350-248	248	to forget	忘記	1523	FORGET
+Zhou-2020-350-249	249	to fry	油炸	2162	STIR FRY
+Zhou-2020-350-250	250	to get	得到	694	GET
+Zhou-2020-350-251	251	to give, to feed	給、送、餵	1447	GIVE
+Zhou-2020-350-252	252	to go pass	過 v.		
+Zhou-2020-350-253	253	to grip	握	1448	HOLD
+Zhou-2020-350-254	254	to grow	生長	479	GROW
+Zhou-2020-350-255	255	to grow fruit	結（果子）		
+Zhou-2020-350-256	256	to grow up	長大	552	GROW UP
+Zhou-2020-350-257	257	to hang	掛	2884	HANG
+Zhou-2020-350-258	258	to hear	聽見	1408	HEAR
+Zhou-2020-350-259	259	to hold a crutch	拄		
+Zhou-2020-350-260	260	to infect	傳染		
+Zhou-2020-350-261	261	to itch	癢	3856	ITCH OR ITCHY OR ITCHING
+Zhou-2020-350-262	262	to laugh	笑	1355	LAUGH
+Zhou-2020-350-263	263	to lay (egg)	下（蛋）	2071	LAY (VERB)
+Zhou-2020-350-264	264	to learn	學	504	LEARN
+Zhou-2020-350-265	265	to lend	借出	582	LEND
+Zhou-2020-350-266	266	to lend out	借出		
+Zhou-2020-350-267	267	to lick	舔	319	LICK
+Zhou-2020-350-268	268	to lie down, to lower one's head	低頭、躺	215	LIE DOWN
+Zhou-2020-350-269	269	to lie on one's belly	趴著、孵 ༴ 'to lie down'		
+Zhou-2020-350-270	270	to light up	點燃	737	LIGHT (IGNITE)
+Zhou-2020-350-271	271	to love	愛	923	LOVE
+Zhou-2020-350-272	272	to marry (receive)	娶、取	2164	MARRY (AS MAN)
+Zhou-2020-350-273	273	to measure	量	1506	MEASURE
+Zhou-2020-350-274	274	to need, to owe	需要、欠	2339	NEED (SOMETHING)
+Zhou-2020-350-275	275	to offer (to god)	敬		
+Zhou-2020-350-276	276	to open	打開	1156	OPEN
+Zhou-2020-350-277	277	to paste	貼		
+Zhou-2020-350-278	278	to permeate	滲入		
+Zhou-2020-350-279	279	to pestle	碓	225	PESTLE
+Zhou-2020-350-280	280	to pick (flower)	摘	2148	PICK
+Zhou-2020-350-281	281	to pick up	撿	371	PICK UP
+Zhou-2020-350-282	282	to plant, to stick in	插、栽	1486	PLANT (SOMETHING)
+Zhou-2020-350-283	283	to pound	舂	354	POUND
+Zhou-2020-350-284	284	to press	壓	1913	PRESS
+Zhou-2020-350-285	285	to puke	嘔吐	1278	VOMIT
+Zhou-2020-350-286	286	to pull	拔	1455	PULL
+Zhou-2020-350-287	287	to rain	下（雨）	1253	RAIN (RAINING)
+Zhou-2020-350-288	288	to release	放	1095	LET GO OR SET FREE
+Zhou-2020-350-289	289	to rest	休息	168	REST
+Zhou-2020-350-290	290	to rise (water)	漲		
+Zhou-2020-350-291	291	to roast	烤	1962	ROAST (SOMETHING)
+Zhou-2020-350-292	292	to roll	滾	827	ROLL
+Zhou-2020-350-293	293	to scrape	刮	1767	SCRAPE
+Zhou-2020-350-294	294	to see	看見	1409	SEE
+Zhou-2020-350-295	295	to send away	送（走）		
+Zhou-2020-350-296	296	to sew	縫	1457	SEW
+Zhou-2020-350-297	297	to sit	坐	1649	SIT DOWN
+Zhou-2020-350-298	298	to snow	下（雪）	2234	FALL (OF SNOW AND RAIN)
```

Comment:
1255 SNOWING ?

Reply:
difficult case here: the verb means clearly "fall (of things from sky)", but the English translation is giving "to snow". I think we should leave it as is, since the Concepticon Gloss "FALL (OF SNOW AND RAIN)" was made for this very verb 下 and I'd say the English translation is problematic given by the dataset, but that should not bother us.